app/assets/stylesheets/local/custom.scss

```diff
@@ -1,2 +1,69 @@
 // App-specific custom styles and overrides
 //
+
```

Comment:
Maybe we can move all this to their own file inside local/ and then reference the import the file in the application.scss, as it all relate to the same thing (flash card).

I've not tested it as I'm in the middle of something else but a bit worried there might be something else to consider here (apart from ideally not hardcoding colours, paddings, margins, etc. and use the GOV.UK values?).

The one we did for C100 (inspired by BEIS) can serve as an example (but we probably don't need everything in there as we don't have chevron buttons, etc.)

https://github.com/ministryofjustice/c100-application/blob/master/app/assets/stylesheets/local/intro_card.scss